docs/goval-dictionary.md

```diff
@@ -26,7 +26,6 @@ $ vuls report
 
 ## Usage: Update OVAL Data
 
-- [Red Hat, CentOS, AlmaLinux, Rocky Linux](https://github.com/vulsio/goval-dictionary#usage-fetch-oval-data-from-redhat)
```

Comment:
Although vuls is not used, goval-dictionary can still fetch redhat's OVAL, so there may be no need to delete it from here.

Reply:
Well spotted.  Fetching redhat oval data gives (almost) no benefit for users (other than investigating the data by themselves.)  I will go as is. Thanks!